docs/operation/cluster_and_service_configuration.rst

```diff
@@ -147,6 +148,45 @@ To add a new static user, perform the following operations:
 #. Verify that the user has been successfully added and you can log in to the
    MetalK8s UI using the new email and password.
 
+.. _Change-dex-user-password:
+
+Change Password for local static user
+"""""""""""""""""""""""""""""""""""""
+
+To change the password of an existing user, perform the following operations:
+
+#. Generate a bcrypt hash of the new password.
+
+   .. code-block:: shell
+
+      root@bootstrap $ htpasswd -bnBC 14 "" <replace-with-password> | tr -d ':'
+
+#. From the Bootstrap node, edit the ConfigMap ``metalk8s-dex-config`` and then
+   change the ``hash`` for the selected user:
```

Comment:
Maybe show an example of the diff on a changed ConfigMap, to make it easier for users?